pyproject.toml

```diff
@@ -16,55 +15,67 @@ classifiers = [
   "Topic :: Software Development",
   "Intended Audience :: Developers",
   "Typing :: Typed",
+  "License :: OSI Approved :: MIT License",
+]
+requires-python = ">=3.11"
+dependencies = [
+  "anyio>=4.3.0",
```

Comment:
Does this include anyio 5.x.x now as well? With the old Poetry `^4.3.0` version specification, the version would be constrained to `<5.0.0`. Similarly for all other dependencies.

Reply:
yes, all upper constraints are dropped. The uv maintainers have compiled a list of reasons why this is best practice for the Python package ecosystem https://github.com/zanieb/poetry-relax#references

Reply:
Ok, I just imagine it to be a hassle to update dependencies in practice 😅 Will try to read a bit in the links from poetry-relax